src/main/java/de/metas/ui/web/letter/LetterRestController.java

```diff
@@ -134,10 +149,53 @@ public JSONLetter getLetter(@PathVariable("letterId") final String letterId)
 		return JSONLetter.of(letter);
 	}
 
+	@Builder(builderMethodName = "fromLetterBuilder")
+	private I_C_Letter createPersistentLetter(final String subject, final String content)
+	{
+		final I_C_Letter persistentLetter = InterfaceWrapperHelper.newInstance(I_C_Letter.class);
+		persistentLetter.setLetterSubject(subject);
+		persistentLetter.setLetterBody(content);
+		persistentLetter.setLetterBodyParsed(content);
+		InterfaceWrapperHelper.save(persistentLetter);
+		return persistentLetter;
+	}
+
+	private I_C_Letter updatePersistentLetter(final WebuiLetter letter)
```

Comment:
this shall go to WebuiLetterRepository. the I_C_Letter shall not be used outside of repository